test/team_test.rb

```diff
@@ -0,0 +1,73 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/player.rb'
+require './lib/team.rb'
+
+
+class TeamTest < Minitest::Test
+
+  def test_team_exists
+    france = Team.new("France")
+    assert_instance_of Team , france
+
+  end
+
+  def test_it_has_a_country
+    france = Team.new("France")
+    assert_equal "France" , france.country
+  end
+
+  def test_team_has_players
+    france = Team.new("France")
+
+    assert_equal [], france.players
+
+  end
+
+  def test_elminated_default_is_false
+    france = Team.new("France")
+
+    assert_equal false , france.eliminated?
+
+  end
+
+  def test_that_team_can_be_eliminated
+    france = Team.new("France",eliminated = true)
```

Comment:
Careful here! The interaction pattern doesn't allow you to pass `eliminated` as an argument.

Also, while this works, I don't think that it's doing what you think it's doing? Using `eliminated = true` *where you define the method* will allow you to set a default value for a parameter. Passing it `eliminated = true` where you call the method (here in your test) is really passing the *return value* of `eliminated = true`, which happens to be `true` because assignment returns the value on the right of the equal sign.

In short, don't use an `=` in the parenthesis where you call your method.